app/api/v1/endpoints/auth.py

```
@@ -485,54 +485,134 @@ async def refresh_token(
+        try:
+            redis = await get_redis_client()
+            if redis and token:
+                # Use longer TTL for blacklist to be safe
+                blacklist_ttl = max(3600, int(auth_service.access_token_expire.total_seconds()))
+                await redis.setex(
+                    f"blacklist:{token}",
+                    blacklist_ttl,
+                    "revoked_during_logout"
+                )
+                logger.debug("Token blacklisted successfully")
+        except Exception as redis_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)